storybook/.playroom/frameComponent.js

```diff
@@ -12,7 +12,7 @@ import "../.storybook/index.css";
 export default function FrameComponent({ children }) {
   return (
     <IntlProvider lang="en">
-      <FormikDecorator>
+      <FormikDecorator showFieldValues={false}>
```

Comment:
This is ok for me, I wouldn't remove it

Reply:
Playroom doesn't really provide any functionality and is primarily for prototoyping, that's why I don't see the point of having `{}` beneath the rendered components.

Reply:
Playroom doesn't really provide support for functionality and is primarily for prototyping, that's why I don't see the point of having `{}` beneath rendered components.

Reply:
Didn't see that its inside playroom. In that case it's ok